state/s3.yaml

```diff
@@ -425,9 +425,9 @@ Resources:
               }
             }
             if (event.RequestType === 'Delete') {
-              s3.getBucketAcl({
+              s3.send(new GetBucketAclCommand({
                 Bucket: event.ResourceProperties.Bucket
-              }, (err, acl) => {
+              }), (err, acl) => {
                 if (err) {
                   if (err.code === 'NoSuchBucket') {
```

Comment:
sometimes you use err.Code, sometimes stay with err.code?

Reply:
It's `err.name`. :D